internal/app/activity/activity_handler.go

```diff
@@ -137,6 +137,13 @@ func (a *activityHandler) CreateActivity(activity *entity.Activity) (*entity.Act
 		"object": activity,
 	})
 
+	var err error
+	activity.CreatedBy, err = common.GetUserId(a.database, "S0000000")
```

Comment:
We haven't fully discussed how we deal with the activities, but I think this should not always be the System user, but could also be user that can create an activity?

Reply:
Done